tests/system/src/test/java/test/launchertest/Constants.java

```diff
@@ -66,4 +66,6 @@
     static final int ERROR_STARTUP_FAILED = 27;
 
     static final int ERROR_ASSERTION_FAILURE = 28;
+
+    static final int ERROR_TIMEOUT = 28;
```

Comment:
Should not this be assigned error code 29 instead of 28?

Reply:
Yes, thanks for catching this. I fixed it.